lib/Module/KInstruction.cpp

```diff
@@ -18,9 +20,17 @@ KInstruction::~KInstruction() {
   delete[] operands;
 }
 
-void KInstruction::printFileLine(llvm::raw_ostream &debugFile) {
+void KInstruction::printFileLine(llvm::raw_ostream &debugFile) const {
   if (info->file != "")
     debugFile << info->file << ":" << info->line;
-  else
-    debugFile << "[no debug info]";
+  else debugFile << "[no debug info]";
+}
+
+std::string KInstruction::printFileLine() const {
+  if (info->file != "") {
+    std::ostringstream oss;
```

Comment:
I see. Despite that I don't see a reason to deviate from LLVM's coding standards here.